include/common.h

```diff
@@ -52,7 +53,7 @@ bool getSdfParam(sdf::ElementPtr sdf, const std::string& name, T& param, const T
 /**
  * \brief Get a math::Angle as an angle from [0, 360)
  */
-double GetDegrees360(const math::Angle& angle) {
+double GetDegrees360(const ignition::math::Angle& angle) {
```

Comment:
@julianoes can we add `using namespace ignition::math`

Reply:
You should not use `using namespace` in header files.

Reply:
Yeah sorry you are right I didn't see that this was being changed on headers only.